src/integrationTest/java/uk/gov/hmcts/reform/roleassignment/controller/RoleAssignmentCreateAndDeleteIntegrationTest.java

```diff
@@ -265,9 +296,14 @@ private Integer getAssignmentRecordsCount() {
 
     @NotNull
     private HttpHeaders getHttpHeaders() {
+        return getHttpHeaders(AUTHORISED_SERVICE);
```

Comment:
why you create this method?

Reply:
This is to provide.a means of injecting different clientId values.  ClientId values are been taken from that header value in the securityutil class.

Reply:
It was  used at some point but it's now removed